etcdserver/server.go

```diff
@@ -1044,6 +1044,7 @@ func (s *EtcdServer) applyEntryNormal(e *raftpb.Entry) {
 		s.consistIndex.setConsistentIndex(e.Index)
 		shouldApplyV3 = true
 	}
+	s.setAppliedIndex(e.Index)
```

Comment:
I think this should be a `defer`? Otherwise it's updated before the apply call (if any).